guides/source/command_line.md

```diff
@@ -62,6 +62,7 @@ If you wish to skip some files from being generated or skip some libraries, you
 | `--skip-javascript`     | Skip JavaScript files                                    |
 | `--skip-hotwire`        | Skip Hotwire integration                                 |
 | `--skip-jbuilder`       | Skip jbuilder gem                                        |
+| `--skip-kamal`          | Skip Kamal gem                                           |
```

Comment:
The official name of the gem is lowercase: https://rubygems.org/gems/kamal (just like jbuilder):
```suggestion
| `--skip-kamal`          | Skip kamal gem                                           |
```

Reply:
It is capitalized here https://kamal-deploy.org

Reply:
I think the name of the gem is used. 
Jbuilder is capitalized here as well, but the gem name still is jbuilder:
https://github.com/rails/jbuilder?tab=readme-ov-file#contributing-to-jbuilder

Reply:
I could have sworn we discussed maintaining this list recently :thinking: